docs/reference/commandline/logs.md

```diff
@@ -29,7 +30,7 @@ The `docker logs --follow` command will continue streaming the new output from
 the container's `STDOUT` and `STDERR`.
 
 Passing a negative number or a non-integer to `--tail` is invalid and the
-value is set to `all` in that case. This behavior may change in the future.
+value is set to `latest` in that case. This behavior may change in the future.
```

Comment:
Omit "This behavior may change in the future" as the facts of that statement are inherent.